.github/workflows/unit-test.yml

```diff
@@ -3,6 +3,8 @@ on:
   pull_request:
     paths:
     - '**.go'
+    - build/makelib/golang.mk
+    - Makefile
```

Comment:
Opted to run unit tests only on changes to `golang.mk` and not everything in `build/*` because I think it's only changes to `golang.mk` that actually affect how unit tests run.